src/xcode/ENA/ENA/Resources/Localization/ru.lproj/Localizable.strings

```diff
@@ -0,0 +1,888 @@
+
+"ExposureDetection_Explanation_Text_High" = "У Вас повышенный риск заражения, поскольку два дня назад Вы имели продолжительный близкий контакт минимум с одним лицом с положительным тестом на COVID-19.\n\nПоэтому ваш риск заражения был оценен как повышенный.\nРиск заражения рассчитывается по локальным данным определения контакта  (продолжительности и близости) на Вашем устройстве. Данные риска заражения не показываются и не передаются другим лицам.\nПо возвращении домой избегайте близкого контакта с членами вашей семьи.";
+
+"ExposureDetection_Button_Enable" = "Активировать определение контакта";
+
+"ExposureDetection_Button_Refresh" = "Обновить";
+
+/* Exposure Detection Errors */
+"ExposureDetectionError_Alert_Title" = "Что-то пошло не так.";
+
+"ExposureDetectionError_Alert_Action_Details" = "Сведения";
+
+"ExposureDetectionError_Alert_Message" = "Ошибка при определении контакта.";
+
+/* Settings */
+"Settings_StatusActive" = "Вкл.";
+
+"Settings_StatusInactive" = "Выкл.";
+
+"Settings_StatusEnable" = "Активировать";
+
+"Settings_StatusDisable" = "Деактивировать";
+
+"Settings_KontaktProtokollStatusActive" = "Активно";
+
+"Settings_KontaktProtokollStatusInactive" = "Остановлено";
+
+"Settings_Tracing_Label" = "Определение контакта";
+
+"Settings_Notification_Label" = "Уведомления";
+
+"Settings_Reset_Label" = "Сбросить приложение";
+
+"Settings_Tracing_Description" = "Разрешите генерацию и предоставление случайных идентификаторов COVID-19.";
+
+"Settings_Notification_Description" = "Разрешите автоматические уведомления о Вашем статусе риска заражения COVID-19.";
+
+"Settings_Reset_Description" = "Удалить все Ваши данные в приложении.";
+
+"Settings_NavTitle" = "Настройки";
+
+/* Noticiation Settings */
+"NotificationSettings_On_Title" = "Активировать уведомления о Вашем статусе риска заражения COVID-19?";
+
+"NotificationSettings_On_SectionTitle" = "Настройка";
+
+"NotificationSettings_On_RiskChanges" = "Ваш риск заражения изменился";
+
+"NotificationSettings_On_TestsStatus" = "Статус Вашего теста на COVID-19";
+
+"NotificationSettings_Off_SectionTitle" = "Настройка";
+
+"NotificationSettings_Off_EnableNotifications" = "Уведомления";
+
+"NotificationSettings_Off_StatusInactive" = "Выкл.";
+
+"NotificationSettings_Off_InfoTitle" = "Активировать уведомления";
+
+"NotificationSettings_Off_InfoDescription" = "Чтобы активировать уведомления, нужно разрешить уведомления от приложения Corona-Warn-App в настройках Вашего устройства.";
+
+"NotificationSettings_Off_OpenSettings" = "Открыть настройки";
+
+"NotificationSettings_NavTitle" = "Уведомления";
```

Comment:
https://github.com/corona-warn-app/cwa-app-ios/blob/80beee47a1001beb41284896fc870bec89d95457/src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings#L155-L159
https://github.com/corona-warn-app/cwa-app-ios/blob/80beee47a1001beb41284896fc870bec89d95457/src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings#L155-L159

I think NotificationSettings_onImageDescription  and NotificationSettings_offImageDescription are missing.

Reply:
these texts will come to the Russian translation a bit later this week